tests/test_missing_handler.json

```diff
@@ -4,7 +4,7 @@
   "expected": [
     {
       "status": "error",
-      "content": "{\"errorMessage\" : \"unable to compile handlers.ts\", \"errorType\" : \"InitException\"}"
+      "content": "{\"errorMessage\" : \"deno exited\", \"errorType\" : \"InitException\"}"
```

Comment:
I imagine this is a somewhat subtle bug (in error message) :(
(debugging this is always tricky... so ignoring for now)